File: Java/Account.java
package Java;

public class Account {
    Integer id;
    String name;
    String document;
    String email;
    String password;

    public Account(String name, String document){
        this.name = name;
        this.document = document;
    }

    void printDataAccount(){
        System.out.println("Name: " + name);
        System.out.println("Document: " + document);
        System.out.println("Email: " + email);
        System.out.println("----------------------------");
    }
  
}

File: JS/Account.js
class Account{
    constructor(name, document){
        this.id;
        this.name = name;
        this.document = document;
        this.email;
        this.password;
    }
}

Account.prototype.printDataAccount = function(){
    console.log(this.name)
    console.log(this.document)
    console.log(this.email)
}